src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = {};
+  const propertyValuePairs = sourceString.split(';');
+
+  for (const pair of propertyValuePairs) {
+    const [property, value] = pair.split(':').map((str) => str.trim());
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable.


BAD EXAMPLE:
```
const cssStyles = styles.map(style => style).filter(style => style.length).slice(0, 5)
```

GOOD EXAMPLE: 
```
const cssStyles = styles
 .map(style => style)
 .filter(style => style.length)
 .slice(0, 5)
```

Please edit the code style used on this line.